src/mc2acab/MCNP_ACAB.py

```diff
@@ -155,6 +169,9 @@ def __parse_args(reqs, options, args):
                                   ' is not here \nEnter outp file name: ')
     if reqs['-st_units'] == '2':
         reqs['-st'] *= 6.24E15
+    if options['-sce_file']:
```

Comment:
Remove from this PR and add it in another commit.

Reply:
OK